torch/_inductor/codegen/cuda/__init__.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+
+THIRD_PARTY_PATH = "../../../../third_party/"
```

Comment:
Nit: This looks quite fragile in the face of possible file structure reorg. Maybe use `torch` module's path as the base?